test/uk/gov/hmrc/agentservicesaccount/controllers/SuspendedJourneyControllerSpec.scala

```diff
@@ -16,40 +16,58 @@
 
 package uk.gov.hmrc.agentservicesaccount.controllers
 
+import com.google.inject.AbstractModule
 import org.jsoup.Jsoup
 import play.api.test.{FakeRequest, Helpers}
 import uk.gov.hmrc.agentservicesaccount.support.{BaseISpec, Css}
 import play.api.http.Status.{BAD_REQUEST, OK, SEE_OTHER}
 import play.api.i18n.{Lang, Messages, MessagesApi}
-import play.api.test.Helpers.defaultAwaitTimeout
+import play.api.test.Helpers.{await, defaultAwaitTimeout}
 import play.twirl.api.HtmlFormat
 import uk.gov.hmrc.agentservicesaccount.config.AppConfig
 import uk.gov.hmrc.http.SessionKeys
 import play.api.http.MimeTypes.HTML
 import play.api.mvc.Result
 import uk.gov.hmrc.agentmtdidentifiers.model.SuspensionDetails
+import uk.gov.hmrc.agentservicesaccount.services.SessionCacheService
 import uk.gov.hmrc.agentservicesaccount.stubs.AgentClientAuthorisationStubs.givenSuspensionStatus
+import uk.gov.hmrc.agentservicesaccount.stubs.SessionServiceMocks
 
 import scala.concurrent.Future
 
-class SuspendedJourneyControllerSpec extends BaseISpec {
+class SuspendedJourneyControllerSpec extends BaseISpec with SessionServiceMocks{
+
+  trait Test{
+
```

Comment:
?

Reply:
you can remove this empty trait @raw1697